content/02.introduction.md

```diff
@@ -1,29 +1,25 @@
-Common preprint repositories include arXiv [@doi:10.1108/14666180010345564], bioRxiv [@doi:10.1101/833400] and medRxiv [@url:https://www.medrxiv.org/]; however, there are over 60 different repositories available [@url:https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/].
-
-The scientific community has begun to analyze the impact of preprints in the life sciences.
-Preprints are being posted at an increasing rate [@doi:10.1371/journal.pbio.3000269].
-Preprints are also rapidly shared on social media, routinely downloaded, and cited [@doi:10.1371/journal.pone.0047523].
-Articles with matching preprint versions are cited and discussed more often than articles without them [@doi:10.1162/qss_a_00043; @doi:10.7554/eLife.52646].
+Preprints are now hosted on online repositories, where users have open and easy access to these works.
+Notable repositories include arXiv [@doi:10.1108/14666180010345564], bioRxiv [@doi:10.1101/833400] and medRxiv [@url:https://www.medrxiv.org/]; however, there are over 60 different repositories available [@url:https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/].
+Previous work that focused on the bioRxiv repository found that life science preprints are being posted at an increase rate [@doi:10.1371/journal.pbio.3000269].
+Furthermore, these preprints are also being rapidly shared on social media, routinely downloaded, and cited [@doi:10.1371/journal.pone.0047523].
 Certain categories of preprints seem to be read and shared differently by both scientists and non-scientists [@doi:10.1101/2020.03.06.981589].
-Across preprint servers, analyses suggest that between two-thirds to three-quarters of preprints are eventually published [@doi:10.7554/eLife.45133.001; @doi:10.1002/pra2.175].
-The time required for a preprint to be published can vary from preprint to preprint; however, preprints with a single version often take less time to publish than preprints with multiple versions [@doi:10.2139/ssrn.3455146], suggesting that authors may update their preprints between submissions for peer review.
+Plus, life science articles with matching preprint versions are cited and discussed more often than articles without them [@doi:10.1162/qss_a_00043; @doi:10.7554/eLife.52646].
+These studies have primarily focused on the metadata behind preprints; however, the textual content contained within preprints has yet to be examined.
 
-Studies of life sciences preprints have primarily focused on the metadata associated with these articles; however, their textual content remains unexamined.
+Preprints have often faced skepticism within the life science community [@doi:10.1371/journal.pgen.1008565; @doi:10.1136/bmj.m3111; @doi:10.1007/s10393-018-1352-3; @doi:10.1073/pnas.1511912112].
+One particular problem focuses on the publication process and the time it takes for preprints to become published.

```

Comment:
Is this a problem, an avenue of inquiry, or something else? Is the problem that some preprints are never published in peer reviewed form? Is that a problem?

I am somewhat missing the connection here.